### DISCUSSION TITLE
Is there a way to make Infolist Tabs Lazy?

### DISCUSSION THREAD

**Discussion by bomshteyn** [filamentphp/filament#7868] [Help]
I have a view page that has many tabs and am trying to make tabs load only when active, or load behind the scenes once the active tab finished loading. Is that doable?

Also when using `View::make()` inside the the infolist schema is there a way to pass params to the view? 
Similar to basic Laravel: `@include('view.name', ['status' => 'complete'])` ?

**Comment by pboivin** [comment]
I'm not sure the `Tabs` component supports lazy loading like this.

You can pass parameters to your view component with `->viewData([ ... ])`.
